ohno.md

```diff
@@ -108,12 +108,16 @@
 
 ... someone converts the country of Canada to a bus stop.
 
+... someone converts a bus stop to a country.
+
```

Comment:
This one would be quite a hard feat to have happen, people won't be trying to make new countries.